sdk/src/main/scala/com/horizen/api/http/ApiResponseUtil.scala

```diff
@@ -26,17 +29,14 @@ object ApiResponseUtil {
   def toResponseAsJava(response: ApiResponse): server.Route = {
     response match {
       case _: SuccessResponse => Directives.complete(StatusCodes.OK, HttpEntities.create(ContentTypes.APPLICATION_JSON, SerializationUtil.serializeWithResult(response)))
-      case e: ErrorResponse =>
-        e.exception match {
-          case Some(thr) =>
-            val msg = thr.getMessage
-            if (msg != null && !msg.isEmpty)
-              Directives.complete(StatusCodes.OK, HttpEntities.create(ContentTypes.APPLICATION_JSON, SerializationUtil.serializeErrorWithResult(e.code, e.description, msg)))
-            else
-              Directives.complete(StatusCodes.OK, HttpEntities.create(ContentTypes.APPLICATION_JSON, SerializationUtil.serializeErrorWithResult(e.code, e.description, "")))
-          case None =>
```

Comment:
Do we need to process case ApiResponse? I guess it's allowed to the developer to nest directly from it avoiding `ErrorResponse` or `SuccessResponse`.

Reply:
I don't think so, any response could be error or success. Success is empty trait thus SDK developer shall just inherit from SuccessResponse